src/SourceBuild/tarball/content/repos/Directory.Build.targets

```diff
@@ -197,6 +200,15 @@
       BuildWithOnlineSources="$(BuildWithOnlineSources)"
       KeepFeedPrefixes="@(KeepFeedPrefixes)" />
 
+    <!-- Temporarily add dotnet6 feed to all NuGet.Configs 
```

Comment:
Is there an issue tracking removing this when we can?

Reply:
Thanks for reminding me.  I added one.  https://github.com/dotnet/source-build/issues/2361